contracts/event/EventFactory.sol

```diff
@@ -16,7 +16,7 @@ contract EventFactory is NRC223Receiver {
         uint amount;
     }
 
-    uint16 private constant VERSION = 5;
+    uint16 private constant VERSION = 6;
```

Comment:
can we create a branch called `version-6` and branch off all these changes into that branch, and merge deploy all together when everything is done.